internal/events/webhooks/webhooks.go

```diff
@@ -60,7 +60,8 @@ type whResponse struct {
 	Body    *fftypes.JSONAny   `json:"body"`
 }
 
-func (wh *WebHooks) Name() string { return "webhooks" }
+func Name() string                { return "webhooks" }
```

Comment:
Struggling to understand why we've stopped this being on the interface - will come back to this after rest of review.